docs/source/Support/bskReleaseNotes.rst

```diff
@@ -56,6 +56,9 @@ Version |release|
 - Added support for Vizard 2.3.0
 - Redirected MuJoCo errors and warnings to :ref:`bskLogging` instead of printing to file.
 - Support calling ``unsubscribe`` on input messages.
+- The ``BSKLogger`` now raises a ``BasiliskError`` (``Basilisk.architecture.bskLogging.BasiliskError``)
+  if something is logged at ``BSK_ERROR`` level. Certain errors are reduced to ``BSK_WARNING`` to reflect
```

Comment:
- add RST documentation on exception handling.  Under Learning BSK, add Advanced RST page.
- add line to known issues that prior code that threw BSK_ERROR warnings will now break.